pkg/security/probe/probe_windows.go

```diff
@@ -369,6 +412,26 @@ func (p *WindowsProbe) approve(field eval.Field, eventType string, value string)
 
 	return false
 }
+func (p *WindowsProbe) auditEtw(ecb etwCallback) error {
+	log.Info("Starting tracing...")
+	err := p.auditSession.StartTracing(func(e *etw.DDEventRecord) {
+
+		switch e.EventHeader.ProviderID {
+
+		case etw.DDGUID(p.auditguid):
+			switch e.EventHeader.EventDescriptor.ID {
+			case idObjectPermsChange:
+				if pc, err := p.parseObjectPermsChange(e); err == nil {
+					log.Infof("Received objectPermsChange event %d %s\n", e.EventHeader.EventDescriptor.ID, pc)
```

Comment:
Any kind of ETW event logging has potential for overloading our own log system, never mind triggering other ETW events, causing potential for never ending indirect loop IMO